src/rules/import-scope.js

```diff
@@ -30,6 +30,40 @@ const isMethodImport = name => getMethodImportFromName(name) && !includes(name,
 const isMethodPackageImport = name => getMethodImportFromName(name) && includes(name, '.')
 const allImportsAreOfType = (node, types) => every(node.specifiers, specifier => includes(types, specifier.type))
 
+/**
+ * Creates a fix(fixer) method for `full` imports
+ * @param {*} context 
+ * @param {*} node 
+ * @returns 
+ */
+const createFullFix = (context, node) => {
+    const sourceCode = context.getSourceCode();
+    const text = sourceCode.getText(node);
+    if ( !/.*(lodash)';$/.test(text) ) {
+      return () => [];
+    }
+
+    const regexp = new RegExp(/(\w+)/g);
+    const matches = text.match(regexp);
+    if ( !matches ) {
+      return () => [];
+    }
+
+    const libs = matches
+    .filter( match => !['import', 'from', 'lodash'].includes(match))
+    if (!libs || libs.length < 1) {
+      return () => [];
+    }
+
+    const imports = libs
+    .sort()
```

Comment:
please fix the formatting

Reply:
Hi there - (no idea what I'm doing) - by "fix the formatting" you mean what exactly?   The `lint` passes.    So, I am not sure what needs to be fixed.   Thank you for any clarification.

Reply:
nvm, I'll fix it

Reply:
Hi @idok - 

1) The .eslintrc.json needed `root: true` in order for the linter to work.   From there, I was able to `lint -- --fix`

2) I added a test suite, `package-to-method` for import-scope.

Let me know if there is more work to be done, or if I am completely off track.   Thank you for your patience.